### PR TITLE
Move methods to util

### LINKED ISSUE #62: refactor calculated values
factor out `getCalculatedValues` based on this discussion:
https://github.com/FormidableLabs/victory/issues/110

### PR DESCRIPTION
**Pull request by boygirl**
This uses methods from Util, and stops using the pattern of attaching calculated values to `this`
closes #62 when finished

### CODE REVIEW

File: src/components/victory-line.jsx [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-import isNumber from "lodash/lang/isNumber";
-import isObject from "lodash/lang/isObject";
 import isUndefined from "lodash/lang/isUndefined";
 import merge from "lodash/object/merge";
 import pick from "lodash/object/pick";
-import max from "lodash/math/max";
-import min from "lodash/math/min";
-import lodashRange from "lodash/utility/range";
-import React, { PropTypes } from "react";
+import React from "react";
```
Comment (knowbody) [inline src/components/victory-line.jsx]: wouldn't it be better to keep:

``` jsx
import React, { PropTypes } from 'react';
```

and then have something like `UtilsPropTypes` in the `victory-util`:

``` jsx
import { Chart, Data, Domain, UtilsPropTypes, Scale } from 'victory-util';
```

usually when I see `PropTypes` I would think of `React.PropTypes`

**Comment by boygirl** [comment]
cc/ @knowbody This is ready for review

File: demo/demo.jsx [hunk 2]
```diff
@@ -3,11 +3,10 @@ import React from "react";
 import {VictoryLine} from "../src/index";
 import _ from "lodash";
 import {VictoryLabel} from "victory-label";
-import d3Scale from "d3-scale";
 
 export default class App extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
```
Comment (knowbody) [inline demo/demo.jsx]: you don't need to pass props here as they are not used in the constructor

**Comment by knowbody** [comment]
@boygirl just this tiny detail, other than that, great PR, thanks!

**Pull request merged**